bin/utils.js

```diff
@@ -130,6 +147,35 @@ class WSSharedDoc extends Y.Doc {
   }
 }
 
+/**
+ * @param {string} docName - the name of the Y.Doc to find or initialize
+ * @param {boolean} gc - whether to allow gc on the doc (applies only when created)
+ */
+function getOrInitDoc(docName, gc = true) {
+  // get doc, create if it does not exist yet
+  return map.setIfUndefined(docs, docName, () => {
+    const doc = new WSSharedDoc(docName)
+    doc.gc = gc
+    if (persistence !== null) {
+      persistence.bindState(docName, doc)
+    }
+    docs.set(docName, doc)
+    return doc
+  })
+}
+
+/**
+ * Gets a Y.Doc by name, whether in memory or on disk
+ * 
+ * @param {string} docName - the name of the Y.Doc to find or create
+ */
+async function getOrCreateDoc(docName, gc = true) {
+  const ydoc = getOrInitDoc(docName, gc)
```

Comment:
`getOrInitDoc` is going to call `bindState` which is calling `mutualSync`. So this function does the same as this function.